`npm run lint` produces an error

**Describe the bug**
Running linting through the npm script does not work currently

**To Reproduce**
Steps to reproduce the behavior:
1. Go to the project
2. Run `npm run lint`
4. See error

```
$ npm run lint

> ndb-core@0.1.1 lint C:\Users\d062651\dev\aam-digital\ndb-core
> ng lint ndb-core ndb-core-e2e --type-check

Unknown option: 'ndb-core-e2e'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ndb-core@0.1.1 lint: `ng lint ndb-core ndb-core-e2e --type-check`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ndb-core@0.1.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Additional context**
Looks like we need to revert https://github.com/Aam-Digital/ndb-core/pull/190/commits/ba1997d584615bcf80f82d1e40d230fca72a2f12.
Also, on lint I get this warning which could be cleaned up: `no-use-before-declare is deprecated. Since TypeScript 2.9. Please use the built-in compiler checks instead.`

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "start": "ng serve --proxy-config proxy.conf.json",
     "build": "ng build",
     "test": "ng test",
-    "lint": "ng lint ndb-core ndb-core-e2e --type-check",
+    "lint": "ng lint ndb-core --type-check",
     "e2e": "ng e2e",
     "compodoc": "npx compodoc -c doc/compodoc_sources/.compodocrc.json"
   },

